website/content/en/preview/concepts/nodepools.md

```diff
@@ -283,7 +281,7 @@ kubelet:
   maxPods: 20
 ```
 
-You can specify the container runtime to be either `dockerd` or `containerd`. By default, `containerd` is used.
+* `containerd` is the default container runtime on the latest EKS versions, the `spec.template.spec.kubelet.containerRuntime` field was dropped.
```

Comment:
Looking at the docs; considering we have dropped the field entirely, I don't see a point in calling out the container runtime anymore. I think it's fine to drop this asterisk statement as well as the next one that says "`containerd` is the only valid container runtime..." WDYT?

Reply:
@jonathan-innis fixed